### ISSUE TITLE
ArTaxOr_1shot Performance  Issue

### ISSUE THREAD

**Issue by jackLee1987** [lovelyqian/CDFSOD-benchmark#50]
Hello! First of all, thank you very much for your excellent work—your paper has been incredibly inspiring to me.

I’m currently trying to reproduce the results of your code, but I’ve encountered a performance discrepancy specifically in the ArTaxOr_1shot setting. Here are the key details:

In my reproduction, the AP for ArTaxOr_1shot is around 15.15 (consistent across multiple runs).
However, this is lower than the 21.0 AP reported in your paper.
Notably, results on other settings (ArTaxOr_5shot, ArTaxOr_10shot) and other datasets match your paper closely, which suggests most of the reproduction pipeline is correct.

To help diagnose, I’ve checked basic reproducibility steps:

I used the official codebase without modifications to core logic.
Datasets were prepared strictly following the instructions in the paper (e.g., data splitting, annotation format).
I used the recommended pre-trained weights and hyperparameters (learning rate, batch size, etc.) as specified.

I wonder if you have made any special adjustments to the details of ArTaxOr_1shot?

If you have any insights or suggestions for further checks,I’d greatly appreciate your guidance.

Thank you again for your time and work! 
Here are my inference execution results:
[07/18 10:59:21 d2.evaluation.evaluator]: Inference done 1338/1383. Dataloading: 0.0008 s / iter. Inference: 0.5984 s / iter. Eval: 0.0002 s / iter. Total: 0.5994 s / iter. ETA=0:00:26
[07/18 10:59:26 d2.evaluation.evaluator]: Inference done 1346/1383. Dataloading: 0.0008 s / iter. Inference: 0.5990 s / iter. Eval: 0.0002 s / iter. Total: 0.6000 s / iter. ETA=0:00:22
[07/18 10:59:31 d2.evaluation.evaluator]: Inference done 1354/1383. Dataloading: 0.0008 s / iter. Inference: 0.5993 s / iter. Eval: 0.0002 s / iter. Total: 0.6003 s / iter. ETA=0:00:17
[07/18 10:59:37 d2.evaluation.evaluator]: Inference done 1363/1383. Dataloading: 0.0008 s / iter. Inference: 0.5996 s / iter. Eval: 0.0002 s / iter. Total: 0.6006 s / iter. ETA=0:00:12
[07/18 10:59:42 d2.evaluation.evaluator]: Inference done 1373/1383. Dataloading: 0.0008 s / iter. Inference: 0.5990 s / iter. Eval: 0.0002 s / iter. Total: 0.6000 s / iter. ETA=0:00:06
[07/18 10:59:48 d2.evaluation.evaluator]: Inference done 1380/1383. Dataloading: 0.0008 s / iter. Inference: 0.5997 s / iter. Eval: 0.0002 s / iter. Total: 0.6007 s / iter. ETA=0:00:01
[07/18 10:59:50 d2.evaluation.evaluator]: Total inference time: 0:13:48.044446 (0.600903 s / iter per device, on 1 devices)
[07/18 10:59:50 d2.evaluation.evaluator]: Total inference pure compute time: 0:13:46 (0.599853 s / iter per device, on 1 devices)
[07/18 10:59:50 d2.evaluation.coco_evaluation]: Preparing results for COCO format ...
[07/18 10:59:50 d2.evaluation.coco_evaluation]: Saving results to output/vitl/artaxor_1shot/inference/coco_instances_results.json
[07/18 10:59:50 d2.evaluation.coco_evaluation]: Evaluating predictions with unofficial COCO API...
Loading and preparing results...
DONE (t=0.14s)
creating index...
index created!
[07/18 10:59:50 d2.evaluation.fast_eval_api]: Evaluate annotation type *bbox*
[07/18 10:59:51 d2.evaluation.fast_eval_api]: COCOeval_opt.evaluate() finished in 0.34 seconds.
[07/18 10:59:51 d2.evaluation.fast_eval_api]: Accumulating evaluation results...
[07/18 10:59:51 d2.evaluation.fast_eval_api]: COCOeval_opt.accumulate() finished in 0.11 seconds.
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.152
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.338
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.119
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.009
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.153
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.356
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.482
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.495
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.075
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.500
[07/18 10:59:51 d2.evaluation.coco_evaluation]: Evaluation results for bbox: 
|   AP   |  AP50  |  AP75  |  APs  |  APm  |  APl   |
|:------:|:------:|:------:|:-----:|:-----:|:------:|
| 15.153 | 33.752 | 11.893 | 0.000 | 0.891 | 15.307 |
[07/18 10:59:51 d2.evaluation.coco_evaluation]: target AP50: -1
[07/18 10:59:51 d2.evaluation.coco_evaluation]: base AP50: 0.3375183746483307
[07/18 10:59:51 d2.evaluation.coco_evaluation]: all AP50: 0.3375183746483307
[07/18 10:59:51 d2.evaluation.coco_evaluation]: target AP75: -1
[07/18 10:59:51 d2.evaluation.coco_evaluation]: base AP75: 0.1189290634615663
[07/18 10:59:51 d2.evaluation.coco_evaluation]: all AP75: 0.1189290634615663
[07/18 10:59:51 d2.evaluation.coco_evaluation]: target mAP: -1.0
[07/18 10:59:51 d2.evaluation.coco_evaluation]: base mAP: 0.15153370858736964
[07/18 10:59:51 d2.evaluation.coco_evaluation]: all mAP: 0.15153370858736964
[07/18 10:59:51 d2.evaluation.coco_evaluation]: Per-category bbox AP: 
| category   | AP     | category    | AP     | category    | AP     |
|:-----------|:-------|:------------|:-------|:------------|:-------|
| Araneae    | 17.687 | Coleoptera  | 3.758  | Diptera     | 8.984  |
| Hemiptera  | 8.389  | Hymenoptera | 11.560 | Lepidoptera | 26.365 |
| Odonata    | 29.331 |             |        |             |        |
[07/18 10:59:51 d2.engine.defaults]: Evaluation results for ArTaxOr_test in csv format:
[07/18 10:59:51 d2.evaluation.testing]: copypaste: Task: bbox
[07/18 10:59:51 d2.evaluation.testing]: copypaste: AP,AP50,AP75,APs,APm,APl
[07/18 10:59:51 d2.evaluation.testing]: copypaste: 15.1534,33.7518,11.8929,0.0000,0.8907,15.3075
[07/18 10:59:51 d2.evaluation.testing]: ###################### ('AP', 15.153370858736965) ######################

**Comment by yikousige** [comment]
I also have this problem

**Comment by yikousige** [comment]
Can I ask how you solved it later? I’d really appreciate it if you could tell me.
